pkg/version/release_test.go

```diff
@@ -42,12 +42,24 @@ func TestReleaseVersion(t *testing.T) {
 			expectReleaseVersion: "v1.7.0-alpha.1",
 			expectError:          false,
 		},
+		{
+			name:                 "nil version",
+			gitVersion:           "",
+			expectReleaseVersion: "<nil>",
+			expectError:          false,
+		},
 	}
 
 	for i := range tests {
 		tc := tests[i]
 		t.Run(tc.name, func(t *testing.T) {
-			rv, err := ParseGitVersion(tc.gitVersion)
+			var rv *ReleaseVersion
+			var err error
+			if tc.gitVersion != "" {
+				rv, err = ParseGitVersion(tc.gitVersion)
+			} else {
+				rv = &ReleaseVersion{Version: nil}
+			}
```

Comment:
This use case may not be appropriate here because an empty string is not valid for the ParseGitVersion method. The expectError parameter is not considered in one dimension.

Reply:
Thank you for your review, I added that check due to a new test case where r.Version is nil and I overlooked the expectError parameter in that case.